docs/docs/guides/login-session.mdx

```diff
@@ -192,3 +192,9 @@ $ curl -s -X POST -H  "Accept: application/json" \
   }
 }
 ```
+
+## Revoke Sessions using Self-Service
+
+## List Sessions using Self-Service
+
+## List and Revoke Sessions as an Administrator
```

Comment:
@zepatrik could you add a little bit of documentation on how to manage sessions here? Thank you :)